README.md

```diff
@@ -36,14 +36,17 @@ every PR is tested on Ubuntu, Windows and MacOS.
 
 **Scala 2.13, 2.12, 2.11 and Scala 3**. Metals supports these Scala versions:
 
-- **Scala 2.13**: 2.13.8, 2.13.7, 2.13.6, 2.13.5, 2.13.4, 2.13.3, 2.13.2, 2.13.1
+- **Scala 2.13**:
+  2.13.8, 2.13.7, 2.13.6, 2.13.5, 2.13.4, 2.13.3, 2.13.2, 2.13.1
 
-- **Scala 2.12**: 2.12.15, 2.12.14, 2.12.13, 2.12.12, 2.12.11, 2.12.10, 2.12.9,
-  2.12.8
+- **Scala 2.12**:
+  2.12.15, 2.12.14, 2.12.13, 2.12.12, 2.12.11, 2.12.10, 2.12.9, 2.12.8
 
-- **Scala 2.11**: 2.11.12
+- **Scala 2.11**:
+  2.11.12
 
-- **Scala 3**: 3.1.2-RC1, 3.1.1, 3.1.0, 3.0.2, 3.0.1, 3.0.0
+- **Scala 3**:
+  3.1.3-RC2, 3.1.2, 3.1.2-RC3, 3.1.2-RC2, 3.1.1, 3.1.0, 3.0.2, 3.0.1, 3.0.0
```

Comment:
@tgodzik should I remove `3.1.2-RC3, 3.1.2-RC2`?

Reply:
There're still supported so no need for it. We can remove it in Metals though later on.